engine/runtime/src/main/scala/org/enso/interpreter/runtime/IrToTruffle.scala

```diff
@@ -1963,16 +1962,6 @@ class IrToTruffle(
               "a Self occurence must be resolved"
             ).target
           )
```

Comment:
By removing handling of Special names here, and removing `NewRefNode` and co., you are making `AutomaticParallelism` IR pass even more obsolete than it was before (which seems like a good thing because I am convinced that AutomaticParallelism has never actually done anything and will not even work if enabled). It is worth thinking about getting rid of that pass, along with the `Name.Special` IR definitions altogether.

Reply:
OK.